posthog/api/error_tracking.py

```diff
@@ -30,6 +35,57 @@ def safely_get_object(self, queryset) -> QuerySet:
     @action(methods=["POST"], detail=True)
```

Comment:
commenting here so we can thread...

if we only want to check assignee and status then my guess is we need to override the POST and check for the presence of those fields...

might be better to do that in a serializer, i'm not actually 100% sure 🙈

Reply:
We should be able to implement it and just call super, right?